public/theme/skeleton/partials/_recordfooter.twig

```diff
@@ -34,13 +34,19 @@
         </p>
     {% endif %}
 
-    {# @todo See Github Issue https://github.com/bolt/four/issues/252 #}
-    {% set relatedrecords = related_content(record) %}
-    {% if relatedrecords is not empty %}
+    {% set related_cts = record|related_all %}
```

Comment:
Not sure if I like `related_cts`. Why not simply use `related_content` ?

Reply:
Renamed to related_content_types